src/functions/tarsUpload/domain/upload/TARSPayloadConverter.ts

```diff
@@ -129,18 +131,64 @@ export class TARSPayloadConverter implements ITARSPayloadConverter {
     };
   }
 
+  /**
+   * Add TestResult to payload if test is SC
+   * @param completedTestPayload
+   * @param category
+   * @param test
+   * @private
+   */
+  private populateStandardCheckTestResult(
+    completedTestPayload: CompletedTestPayload,
+    category: TestCategory,
+    test: TestResultSchemasUnion,
+  ): CompletedTestPayload {
+
+    if (category === TestCategory.SC) {
+      return {
+        ...completedTestPayload,
+        testResult: this.getStandardCheckTestResult(
+          completedTestPayload.passResult,
+          test?.testData as CatADI3TestData,
+          test?.journalData?.candidate?.previousADITests,
+          test.activityCode),
+      };
+    }
+    return completedTestPayload;
+  }
+
   /**
    * Calculate total assessment score of an ADI3 test
    * @param testData
    */
-  private getTotalAssessmentScore = (testData: CatADI3TestData) : number => {
-    return Object.keys(testData).reduce((sum, key: string) : number => {
+  private getTotalAssessmentScore = (testData: CatADI3TestData): number => {
+    return Object.keys(testData).reduce((sum, key: string): number => {
       const value = get(testData, key);
       if (['lessonPlanning', 'riskManagement', 'teachingLearningStrategies']
         .includes(key) && typeof value === 'object') {
         return sum + (get(value, 'score') || 0);
       }
       return sum;
-    },                                  0);
+    }, 0);
+  };
+
+  /**
+   * Determine the test result for standard checks
+   * @param passResult
+   * @param testData
+   * @param previousADITests
+   * @param activityCode
+   */
+  private getStandardCheckTestResult = (
+    passResult: boolean,
+    testData: CatADI3TestData,
+    previousADITests: number | undefined,
+    activityCode: ActivityCode): string => {
+    if (activityCode === '4') {
+      return 'DAN';
+    }
+    const currentAdiAttempt = previousADITests ? previousADITests + 1 : 1;
+    const isAutomatic = get(testData, 'riskManagement.score', 0) < 8 ? 'A' : '';
+    return passResult ? 'P' : 'F'.concat(isAutomatic, String(currentAdiAttempt));
   };
```

Comment:
If passResult is true, we probs dont need to run any more of the logic, so could we slightly change it to be something like ...
```
if (passResult) return 'P';
if (activityCode === '4') return 'DAN'; // Fail for public safety

const currentAdiAttempt = previousADITests ? previousADITests + 1 : 1;
const isAutomatic = get(testData, 'riskManagement.score', 0) < 8 ? 'A' : '';
return 'F'.concat(isAutomatic, String(currentAdiAttempt));
```